tests/Oryx.BuildImage.Tests/Php/PhpDynamicInstallationTest.cs

```diff
@@ -27,8 +27,10 @@ public PhpDynamicInstallationTest(ITestOutputHelper output) : base(output)
             {
                 var data = new TheoryData<string, string>();
                 data.Add("7.3", ImageTestHelper.WithRestrictedPermissions().GetGitHubActionsBuildImage());
+                data.Add("7.3", ImageTestHelper.WithRestrictedPermissions().GetGitHubActionsBuildImage("buster"));
```

Comment:
I think here should be like `.GetGitHubActionsBuildImage(_gitHubActionsBuster)`, use the constant created in the ImageTestHelper.